### DISCUSSION TITLE
See planet orbits to correct scale in Stellarium 21.3

### DISCUSSION THREAD

**Discussion by Deepskykelly** [Stellarium/stellarium#2369] [Q&A] [answered]
Hello. I am in solar-system-viewer mode and see the orbits of the inner planets greatly exaggerated and the orbits of outer planets extremely condensed. How do I set the planet orbits to the correct scale? Also, my view of the solar system is off center. I've tried moving the sky around, but everything appears highly elliptical no matter what I do. Is there a way to center above the sun?
<img width="816" alt="Screen Shot 2022-03-28 at 7 04 34 AM" src="https://user-images.githubusercontent.com/102528596/160403893-4b7a4825-8953-4865-80cc-9c8503587c64.png">

**Comment by gzotti** [comment]
Please try Alt+cursor keys to move the view. I know there is some more work to do here, but there are always more urgent things.

**Reply by StefanReissl** [reply]
Someone has obviously changed the position of the solar-system-viewer. If you click the sun, you can see that the distance is 1AU. The distance was about 1600AU in an older version of Stellarium.
Can someone fix this issue please? I'm missing the correct scaled orbits, too.

**Reply by gzotti** [reply]
Someone was me, about two years ago. To repeat myself, I know there is some more work to do here, but there are always more urgent things. For simple solar system views, there are several online sites and other tools available. For those things Stellarium can simulate best, there are probably no alternatives. 

Alt+cursor keys move the viewer around the observed object. Alt+End/Home increase/decrease distance. However, orientation is messed up in ways hard to describe. This whole orientation handling class needs a thorough rewrite to make the solar system observers really useful.

**Comment by candylandchampion** [comment]
In the previous version I could center on the Sun using 
option-up arrow on mac 
or 
Alt-up arrow on PC.
Then change the distance using 
Fn-Option-right (or left) arrow on mac 
or 
Alt-END (or HOME) on PC 
to move to a good distance.  With just a 20 AU distance I was able to have students find accurate dates for conjunctions, oppositions, and greatest elongations out to Saturn.

In v1.2, these shortcuts DO NOT work anymore, not even the distance one.  I am at a loss how to do this now and have their lab on this in 2 weeks.

**Reply by gzotti** [reply]
See the previous answer. If your "previous version" was from 2018 and you preferred that you must use that one. Just that you were at a fixed distance and could only zoom (change field of view, not distance). The current manipulation puts you in a gimbal around the sun where you can rotate and change distance with the Alt+Cursor and Alt+End/Home keys described above.  We just have no good solution for the screen orientation.

**Comment by candylandchampion** [comment]
Update to this issue for those that stumble into here looking for answers, V22.2 was working with these commands but now they do not as of summer 24.  Running the newest version, Solar system observer does place you about 30 AU away from the Sun.  However, it is still not centering on the Sun so the orbits are still wonky.  The trick of option - up arrow on mac and Alt- up arrow on PC still does not work.

**Reply by candylandchampion** [reply]
Edit:  Pls see responses below

**Comment by gzotti** [comment, marked as answer]
I just switched to SSO. Location is exactly 30AU from the Sun, and above the ecliptical plane, but not polar. Sun is selected and centered. Orbit lines appear distorted by perspective - 30AU is approximately Neptune's orbit distance.
![stellarium-112](https://github.com/user-attachments/assets/c8d484ca-bfc8-4ee7-a824-8bb43da06ded)
Alt-UpArrow moves to ecliptic north pole. 
![stellarium-111](https://github.com/user-attachments/assets/e312de59-f2bc-4313-b406-5f636839f628)
Other cursor keys (listed in the User Guide) move location elsewhere. 

Please define "wonky".

**Reply by candylandchampion** [reply]
Sorry for delay, went on vacation.  
I completely uninstalled and reinstalled and now it DOES WORK as you say
HUZZAH!

Changing to be over the Sun: 
        option-up arrow on mac  or  Alt-up arrow on PC.
Then change the distance using
        Fn-Option-right (or left) arrow on mac or Alt-END (or HOME) on PC
